### PR TITLE
Refactor request to use await properly

### PR DESCRIPTION
**Pull request by scwe**
If we are going to use ES7 we should probably be consistent.

### CODE REVIEW

File: src/utils/request/request.js [hunk 1]
```diff
@@ -8,36 +8,25 @@ const DEFAULT_OPTIONS = {
...
-        resolve(response)
-      })
-      .catch((err) => {
-        log('err', err)
-        reject(err)
-      })
-  })
+  if (!url) {throw new Error('No URL provided')}
+
+  const requestResult = await fetch(url, options)
```
Comment (wojciechAdaszynski) [inline src/utils/request/request.js]:
```js
if (bla) { 
  throw new Error('No URL provided')
}
```
or at least
```js
{ throw new Error('No URL provided') }
```

Comment (wojciechAdaszynski) [inline src/utils/request/request.js]: you should wrap `fetch` in try...catch block to get error and handle it properly

Comment (scwe) [inline src/utils/request/request.js]: The other PR for standard addition should fix that.

Comment (scwe) [inline src/utils/request/request.js]: What does handle it properly mean? Swallow it? Log it? What? I personally think a stack trace is the most useful thing that can be reported here. Anything else just leads to confusion, especially in developement.

Comment (psmarcin) [inline src/utils/request/request.js]: I agree with @scwe but we also should log it or have a function to log it and pass to our service that will help us track errors

**Comment by psmarcin** [comment]
I'm going to merge this PR and we will create new one for `try` `catch`

**Pull request merged**